### PR TITLE
Fix test password generation rule to meet Windows complexity requirements

### PR DESCRIPTION
**Pull request by SteveL-MSFT**
# PR Summary

The Windows password complexity rule (which is enabled in AzDevOps) requires a minimum 8 character password that contains at least 3 of 5 buckets:

1. number
2. lowercase
3. uppercase
4. symbol
5. unicode

We can skip the unicode character and just pick a random character out of the first 4 buckets twice to have a password meeting the requirement.
The previous code was completely random so it would randomly generate a password that didn't meet the complexity requirement and thus fail to create the test account causing subsequent tests to fail.

## PR Context

Nightly Windows build broke.

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the title (the `WIP` bot will keep its status check at `Pending` while the prefix is present) and remove the prefix when the PR is ready.
- **[Breaking changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)**
    - [x] None
    - **OR**
    - [ ] [Experimental feature(s) needed](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/reference/6/Microsoft.PowerShell.Core/About/about_Experimental_Features.md)
        - [ ] Experimental feature name(s): 
- **User-facing changes**
    - [x] Not Applicable
    - **OR**
    - [ ] [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed: 
- **Testing - New and feature**
    - [ ] N/A or can only be tested interactively
    - **OR**
    - [x] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)

### CODE REVIEW

File: test/powershell/Modules/Microsoft.PowerShell.Management/Set-Service.Tests.ps1 [hunk 1]
```diff
@@ -10,8 +10,18 @@ Describe "Set/New/Remove-Service cmdlet tests" -Tags "Feature", "RequireAdminOnW
         }
         if ($IsWindows) {
             $userName = "testuserservices"
-            $Password = ([char[]]([char]33..[char]95) + ([char[]]([char]97..[char]126)) + 0..9 | Sort-Object {Get-Random})[0..12] -join ''
-            $testPass = (New-Object -TypeName Net.NetworkCredential("", $Password)).SecurePassword
+            $numbers = "0123456789"
```
Comment (adityapatwardhan) [inline test/powershell/Modules/Microsoft.PowerShell.Management/Set-Service.Tests.ps1]: Can you move this logic into a [HelpersCommons.psm1](https://github.com/PowerShell/PowerShell/blob/f645cb8bbbab13c447921d75d7e8dbfa6616a687/test/tools/Modules/HelpersCommon/HelpersCommon.psm1#L313)

**Review by adityapatwardhan — CHANGES_REQUESTED** [review]

File: test/powershell/Modules/Microsoft.PowerShell.Management/Set-Service.Tests.ps1 [hunk 2]
```diff
@@ -10,8 +10,18 @@ Describe "Set/New/Remove-Service cmdlet tests" -Tags "Feature", "RequireAdminOnW
...
+            $lowercase = "abcdefghijklmnopqrstuvwxyz"
+            $uppercase = "ABCDEFGHIJKLMNOPQRSTUVWXYZ"
+            $symbols = "!`"#$%&'()*+,-./:;<=>?[\]^_``"
```
Comment (anmenaga) [inline test/powershell/Modules/Microsoft.PowerShell.Management/Set-Service.Tests.ps1]: Are all of these symbols allowed?

File: test/powershell/Modules/Microsoft.PowerShell.Management/Set-Service.Tests.ps1 (hunk 1 above)
Comment (SteveL-MSFT) [reply test/powershell/Modules/Microsoft.PowerShell.Management/Set-Service.Tests.ps1]: Sure

File: test/powershell/Modules/Microsoft.PowerShell.Management/Set-Service.Tests.ps1 (hunk 2 above)
Comment (SteveL-MSFT) [reply test/powershell/Modules/Microsoft.PowerShell.Management/Set-Service.Tests.ps1]: Found Msft doc page with symbols, so I'll use that to be sure.

**Review by adityapatwardhan — APPROVED** [review]

**Comment by adityapatwardhan** [comment]
@PoshChan please remind me in 30 minutes

**Comment by adityapatwardhan** [comment]
Merged this without waiting for 24 hours as this break daily build.

**Comment by PoshChan** [comment]
@adityapatwardhan, this is the reminder you requested 30 minutes ago

**Pull request merged**